app/views/booking_requests/customer.html.erb

```diff
@@ -2,9 +2,7 @@
 0;font-size: 16px;line-height: 1.315789474;">Dear <%= @booking_request.name %>,</p>
 
 <p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Thank you for requesting a
-guidance appointment with Pension Wise, we'll confirm the appointment slot
-within the next two working days.</p>
+0;font-size: 16px;line-height: 1.315789474;">Thank you for requesting a guidance appointment with Pension Wise. We’ll confirm your appointment in the next 2 working days.</p>
```

Comment:
No the content change is fine here, it was more about the line length being reformatted.